File: test_api/test_whisper_stt.js
import { Client } from "@gradio/client";

const response_0 = await fetch("https://github.com/gradio-app/gradio/raw/main/test/test_files/audio_sample.wav");
const exampleAudio = await response_0.blob();

const client = await Client.connect("http://localhost:7860/");
const result = await client.predict("/predict", {
				inputs: exampleAudio,
		task: "transcribe",
});

console.log(result.data);

File: test_api/test_whisper_stt.py
from gradio_client import Client, handle_file

client = Client("http://localhost:7860/")

result = client.predict(
    inputs=handle_file('https://github.com/gradio-app/gradio/raw/main/test/test_files/audio_sample.wav'),
    task="transcribe",
    api_name="/predict"
)

print(result)
